cognite_toolkit/cdf.py

```diff
@@ -431,9 +459,9 @@ def clean(
             if ToolGlobals.failed:
                 print(f"[bold red]ERROR: [/] Failure to clean {LoaderCls.folder_name} as expected.")
                 exit(1)
-    if "auth" in include:
+    if "auth" in include and (directory := (Path(build_dir) / "auth")).is_dir():
         drop_load_resources(
-            LOADER_BY_FOLDER_NAME.get("auth"),
+            AuthLoader.create_loader(ToolGlobals, load="all"),
```

Comment:
Here the "load" var seems a bit wrong, maybe target would be a more appropriate name?

Reply:
Renamed it to `target_scopes`